src/vaadin-grid.html

```diff
@@ -389,7 +389,8 @@
           heightByRows: {
             type: Boolean,
             value: false,
-            reflectToAttribute: true
+            reflectToAttribute: true,
```

Comment:
is the reflecting to attribute related to the fix? I'm wondering whether we need it as an attribute.

Reply:
I assume it is needed as the attribute is used for styling: https://github.com/vaadin/vaadin-grid/blob/1d8c80a93e5da57f12df09a61f259c085df909d4/src/vaadin-grid-styles.html#L41